### ISSUE TITLE
`statocles status` command

### ISSUE THREAD

**Issue by preaction** [preaction/Statocles#516]
It would be really nice to know a few things about the site that Statocles could know:
- The date of the last deploy of the site
- The `--date` value of the last deploy of the site
- Other things in the future...

To make this happen, when `statocles deploy` is run it should save a `status.yml` file in the `.statocles` directory. This YAML file should contain the following keys:
- `last_deploy_date` - The date/time of the last deploy
- `last_deploy_args` - A hash of arguments to `statocles deploy`, with the following keys:
  - `date` - The `--date` option to the deploy command

Once this file is written, running `statocles status` should display some human-readable text explaining the status of the site. For example, if I last ran the command `statocles deploy --date 2016-06-26` on 2016-10-16 at 2:34 PM, the status command should tell me:

```
$ statocles status
Last deployed on 2016-10-16 at 14:34
Deployed up to date 2016-06-26
```

In the future, a lot more could be done with this, but for now this is the information I need to know to be able to manage my sites.

**Comment by preaction** [comment]
This is being implemented by @perlancar in PR #527

**Comment by preaction** [comment]
This was released with v0.083
